server/server_test.go

```diff
@@ -72,7 +84,109 @@ var _ = Describe("Server", func() {
 				Expect(response.Body.String()).To(Equal("Hello World"))
 			})
 		})
+
+		// TODO: Needs to fix this.
+		/*
+		Describe("API /login", func() {
+			BeforeEach(func() {
+				    body := new(bytes.Buffer)
+					body.Write([]byte(`{"U_Name": "test3", "Password": "test3"}`))
+					response = performRequest(router, "POST", "/login", body)
+					fmt.Println((response.Body.String()))
+				//bytes.NewBuffer([]byte(body))
+			})
+
+			It("Returns with Status 200", func() {
+				Expect(response.Code).To(Equal(200))
+			})
+
+		})
+		*/
+
+
+		Describe("API /dashboard/:id/", func() {
+			BeforeEach(func() {
+				body := new(bytes.Buffer)
+				response = performRequest(router, "GET", "/dashboard/1/", body)
+				fmt.Println((response.Body.String()))
+				//bytes.NewBuffer([]byte(body))
+			})
+
+			It("Returns with Status 200", func() {
+				Expect(response.Code).To(Equal(200))
+			})
+			It("Returns with Output 'Bharath'", func() {
+				Expect(response.Body.String()).To(Equal("\"Bharath\""))
+			})
+
+		})
+
+		Describe("API /dashboard/:id/mylistings", func() {
+			BeforeEach(func() {
+				body := new(bytes.Buffer)
+				response = performRequest(router, "GET", "/dashboard/1/mylistings", body)
+				fmt.Println((response.Body.String()))
+				//bytes.NewBuffer([]byte(body))
+			})
+
+			It("Returns with Status 200", func() {
+				Expect(response.Code).To(Equal(200))
+			})
```

Comment:
Can't we assert against the data that we received? I think that will be a better test than just checking the status!